### PR TITLE
topology: sof-jsl-da7219: Add KWD support

### PR DESCRIPTION
**Pull request by yongzhi1**
Pipelines added for Keyphrase Detection feature support.

Signed-off-by: Yong Zhi <yong.zhi@intel.com>

### CODE REVIEW

**Comment by yongzhi1** [comment]
The above changes have not been tested yet, just want to make sure they do not have obvious errors. cc @keyonjie

**Comment by xiulipan** [comment]
@yongzhi1 can this tplg loaded on platform? Is the buffer enough?

File: tools/topology/sof-jsl-da7219.m4 [hunk 1]
```diff
@@ -153,6 +176,30 @@ PCM_PLAYBACK_ADD(HDMI1, 2, PIPELINE_PCM_5)
...
+	PIPELINE_SCHED_COMP_8,
+	SCHEDULE_TIME_DOMAIN_TIMER,
+	16000, 16000, 16000)
+
+dnl P_GRAPH(NAME, PIPELINE_ID, CONNECTIONS)
+SectionGraph."pipe-sof-jsl-keyword-detect" {
+	index "0"
+
+	lines [
+dnl create direct DAPM/pipeline link between 2 widgets)
```
Comment (lgirdwood) [inline tools/topology/sof-jsl-da7219.m4]: Any reason why we are keeping these as comments ?

Comment (lgirdwood) [reply tools/topology/sof-jsl-da7219.m4]: No, different audio direction.

**Comment by keyonjie** [comment]
the rebased version looks good to me, in case the KWD can work for you. @yongzhi1

**Comment by lgirdwood** [comment]
CI known issues.

**Pull request merged**